parquet-encoding/src/main/java/org/apache/parquet/bytes/BytesInput.java

```diff
@@ -164,6 +176,9 @@ public static BytesInput copy(BytesInput bytesInput) throws IOException {
     return baos.getBuf();
   }
 
+  public ByteBuffer toByteBuffer() throws IOException {
+    return ByteBuffer.wrap(toByteArray());
+  }
```

Comment:
we should use the appropriate ByteBuffer.wrap when it's backed by an array with an offset and a length.